modules/instrument_manager/php/NDB_Menu_Filter_instrument_manager.class.inc

```diff
@@ -73,6 +73,7 @@ class NDB_Menu_Filter_instrument_manager extends NDB_Menu_Filter
 
     function _setDataTableRows($count)
     {
+        $config = NDB_Config::singleton();
```

Comment:
The config object should be created with NDB_Factory, not the singleton method.. also, I think the Settings object (which can also be created from the factory...) is a cleaner way of getting this since it directly has a Settings->dbName() function to get the database name